w07h03/test/pgdp/sim/sampleSimulation.java

```diff
@@ -0,0 +1,82 @@
+package pgdp.sim;
+
+import java.util.Random;
+import org.junit.jupiter.api.Test;
+
+public class sampleSimulation {

```

Comment:
```suggestion
public class SampleSimulation {
```

Classes should be capitalized